lib/system.nim

```diff
@@ -2339,7 +2339,7 @@ when notJSnotNims:
   when hostOS != "standalone" and hostOS != "any":
     include "system/dyncalls"
 
-  include "system/sets"
+  from system/sets import cardSet
```

Comment:
This is blocking the merge until my basic IC tests are merged. Yet another system module dependency...

Reply:
ok